src/test/kotlin/calculator/domain/OperandTest.kt

```diff
@@ -0,0 +1,32 @@
+package calculator.domain
+
+import calculator.constants.Messages
+import org.assertj.core.api.Assertions
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+
+/**
+ * Created by Jaesungchi on 2022.05.21..
+ */
+class OperandTest {
+    @Test
+    fun `음수를 전달 받으면 RuntimeException을 throw한다`() {
+        Assertions.assertThatExceptionOfType(RuntimeException::class.java).isThrownBy {
+            Operand.of("-1")
+        }.withMessageMatching(Messages.INSERT_NEGATIVE_NUMBER)
```

Comment:
```suggestion
        assertThrows<RuntimeException> { Operand.of("-1") }
```
assertThrows를 사용하면, 예외 처리에 대해서도 간결하게 테스트 코드를 짤 수가 있습니다.

추가로 assertTatExceptionOfType()과 assertThrows<>의 가장 큰 차이점은 .class.java 파일을 쓸 것인지 .kt 파일을 쓸 것인지에 대한 차이점이 있습니다. 이러한 차이점이 있기 때문에 assertThatExceptionOfType은 java Exception class 만 받을 수 있습니다. 코틀린 Exception class를 받기 위해서도 결국 java로 변경해서 사용을 해야될거에요.
하지만, assertThrows<>는 이미 .kt 파일로 만들어졌기 때문에, kotlin의 익셉션 처리를 할 수 있고, 순수한 kotiln API로 에러 테스트를 할 수 있게 만듭니다.

저의 개인적인 생각이지만, kotlin 코틀린스럽게 짜는 코드는 kotlin 프로젝트에서 자바와 혼용해서 짜는 것이 아닌, kotlin에 있는 API만으로 코드를 짜는 것부터 시작한다고 생각합니다 : )

앞으로 있을 미션에 대해 java API가 아닌, kotlin API로 짜는 연습을 해보시길 추천드립니다! 😁